test/perf/ubench/ubench.cc

```diff
@@ -97,7 +97,7 @@ namespace ubench
         ((pinger->rng.next() % pinger->select_mod) == 0);
```

Comment:
Is the reordering in this file essential?  Just makes it hard to see what has actually changed.